storage/migrations/02_runtimes.up.sql

```diff
@@ -207,6 +207,17 @@ CREATE TABLE chain.evm_token_balance_analysis
 );
 CREATE INDEX ix_evm_token_balance_analysis_stale ON chain.evm_token_balance_analysis (runtime, token_address, account_address) WHERE last_download_round IS NULL OR last_mutate_round > last_download_round;
 
+CREATE TABLE chain.evm_contracts
+(
+  runtime runtime NOT NULL,
+  contract_address oasis_addr NOT NULL,
+  creation_tx HEX64 NOT NULL,
+  bytecode BYTEA,
+  abi JSONB,
+  verified BOOLEAN NOT NULL,
```

Comment:
Small nit: visually separate fields that are only present for verified contracts, something like:
```sql
CREATE TABLE chain.evm_contracts(
  runtime runtime NOT NULL,
  <other fields present for all contracts>
  ...
  verified BOOLEAN NOT NULL 
  -- Fields below are only populated for verifed contracts.
  abi JSONB
...

Reply:
Got it, will update!